sdk/src/main/java/io/opentelemetry/sdk/trace/RandomIdsGenerator.java

```diff
@@ -16,20 +16,18 @@
 
 package io.opentelemetry.sdk.trace;
 
+import java.util.concurrent.ThreadLocalRandom;
```

Comment:
Please also fix this import order - we are strictly following the Google Format guidelines ;)

Reply:
You should be strict :)
I don't know what happened to my IntelliJ google format plugin, it did fix it now during formatting.

Reply:
You can also use the gradle command goJF

Reply:
very cool, thanks